collector/fsrmquota.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func init() {
-	Factories["fsrmquota"] = NewFSRMQuotaCollector
+	Factories["fsrmquota"] = newFSRMQuotaCollector
```

Comment:
The `Factories` dictionary has been removed, so the new registration is like this:
```suggestion
	registerCollector("fsrmquota", newFSRMQuotaCollector)
```